app/controllers/users_controller.rb

```diff
@@ -60,7 +60,7 @@ def change_password
   def show
     @repo_user = User.find_by username: params[:username]
     @github_username = Octokit::Client.new(access_token: @repo_user.access_token).login
-    @repositories = @repo_user.repositories.where(make_id: nil).page params[:page]
+    @repositories = @repo_user.repositories.has_photos.where(make_id: nil).page params[:page]
```

Comment:
nvm I don't like this cus it hides the repositories and it can cause confusion. I'll try the default image tmr